docs/example-react-hooks-useContext.md

```diff
@@ -0,0 +1,92 @@
+This is a simple example to test a component using the useContext hook. 
+
+
+The context object
```

Comment:
I just noticed that other examples add the filename on top of the code snippet block, as in

```js
// context.js

import React from 'react'

....
```

I feel that could be seen as the "title" of the snippet. It'd be great to keep that strategy, to make things consistent. What do you think about it?

Reply:
yeah makes sense